docker-compose.yml

```diff
@@ -11,18 +12,15 @@ services:
       - /dev/dri:/dev/dri # for intel hwaccel, needs to be updated for your hardware
     volumes:
       - /etc/localtime:/etc/localtime:ro
-      - .:/opt/frigate:cached
+      - .:/lab/frigate:cached
       - ./config/config.yml:/config/config.yml:ro
       - ./debug:/media/frigate
-      - type: tmpfs # Optional: 1GB of memory, reduces SSD/SD Card wear
-        target: /tmp/cache
-        tmpfs:
-          size: 1000000000
     ports:
       - "1935:1935"
       - "5000:5000"
       - "5001:5001"
       - "8080:8080"
+    entrypoint: ["sudo", "/init"]
```

Comment:
FYI this produces a error for me and I cannot start the frigate-dev container.
```
sudo: /init: command not found
```

Reply:
Make sure you rebuild frigate:latest

Reply:
Thanks, that was my problem. Sorry for the noise.